### PR TITLE
CreateImageWizard: single upload_request

### PR DESCRIPTION
**Pull request by chloenayon**
Change image_request to accept a single upload_request instead of
an array of upload_requests.

### CODE REVIEW

**Comment by Gundersanne** [comment]
Nice! Adding blocked until the related composer PR is merged (osbuild/osbuild-composer#1267)

File: src/SmartComponents/CreateImageWizard/CreateImageWizard.js
Comment (Gundersanne) [inline src/SmartComponents/CreateImageWizard/CreateImageWizard.js:262]: This needs to not be an array now

File: src/SmartComponents/CreateImageWizard/CreateImageWizard.js
Comment (Gundersanne) [inline src/SmartComponents/CreateImageWizard/CreateImageWizard.js:281]: ditto

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Because of #103  this needs a rebase now

**Comment by chloenayon** [comment]
made fixes to address @Gundersanne 's comments

**Review by Gundersanne — APPROVED** [review]

**Pull request merged**
